application/common/components/Emailer.php

```diff
@@ -19,6 +21,21 @@ class Emailer extends Component
     const SUBJECT_PASSWORD_CHANGED_DEFAULT = 'Your %s account password has been changed';
     const SUBJECT_WELCOME_DEFAULT = 'Welcome to your new %s account';
 
+    const SUBJECT_GET_BACKUP_CODES_DEFAULT = 'Get printable codes for your %s account';
+    const SUBJECT_REFRESH_BACKUP_CODES_DEFAULT = 'Get a new set of printable codes for your %s account';
+    const SUBJECT_LOST_SECURITY_KEY_DEFAULT = 'Have you lost the security key you use with your %s account';
+
+    const SUBJECT_MFA_OPTION_ADDED_DEFAULT = 'A 2-Step Verification option was added to your %s account';
+    const SUBJECT_MFA_OPTION_REMOVED_DEFAULT = 'A 2-Step Verification option was removed from your %s account';
+    const SUBJECT_MFA_ENABLED_DEFAULT = '2-Step Verification was enabled on your %s account';
+    const SUBJECT_MFA_DISABLED_DEFAULT = '2-Step Verification was disabled on your %s account';
+
+    /* The number of days of not using a security key after which we email the user */
```

Comment:
It just occurred to me that, if someone primarily uses their security key (U2F) and has their browser remember them for 30 days, then if they use some other computer (like the one in LSC 108) and use a code because it's more convenient, they'll probably get this email.

We may want to bump this up from 14 days to something noticeably over 30 days. Perhaps 40?

Reply:
@fillup Input on this?

Reply:
I made this a config